cmake/multi_image.cmake

```diff
@@ -185,7 +185,95 @@ function(get_child_image_cmake_args)
 
   unset(${GET_ARGS_CONFIG_OUT} PARENT_SCOPE)
   set(${GET_ARGS_CONFIG_OUT} "${child_image_cmake_args}" PARENT_SCOPE)
+endfunction()
+
+function(add_s1_variant)
```

Comment:
The args here do not appear to be variable.

Reply:
You can choose to not send `BASE_IMAGE`